r/rsparkling/rsparkling.Rproj

```diff
@@ -13,4 +13,4 @@ StripTrailingWhitespace: Yes
 BuildType: Package
 PackageUseDevtools: Yes
 PackageInstallArgs: --no-multiarch --with-keep.source
-PackageRoxygenize: rd,collate,namespace
+PackageRoxygenize: rd,collate
```

Comment:
I do not know what is common R community practice - keep and share, or remove? In Java world we do not share user specific settings...